samples/notebooks/image-captioning-gcp/Image Captioning TF 2.0.ipynb

```diff
@@ -0,0 +1,893 @@
+    "        output_dir=tokenizing_output_dir, \n",
+    "        top_k=vocab_size).apply(\n",
+    "        use_gcp_secret('user-gcp-sa'))\n",
+    "    \n",
+    "    model_train_task = model_train_op(\n",
+    "        dataset_path, \n",
+    "        preprocessing_img_task.output,\n",
+    "        tokenize_captions_task.output,\n",
+    "        train_output_dir=training_output_dir, \n",
+    "        valid_output_dir=validation_output_dir,\n",
+    "        batch_size=training_batch_size, \n",
+    "        embedding_dim=embedding_dim, \n",
+    "        units=hidden_state_size, \n",
+    "        epochs=epochs).apply(\n",
+    "        use_gcp_secret('user-gcp-sa'))\n",
+    "    \n",
+    "    predict_task = predict_op(\n",
+    "        dataset_path,\n",
+    "        tokenize_captions_task.output, \n",
+    "        model_train_task.output,\n",
+    "        preprocess_output_dir=preprocessing_output_dir,\n",
+    "        valid_output_dir=validation_output_dir,\n",
+    "        embedding_dim=embedding_dim,\n",
+    "        units=hidden_state_size).apply(\n",
+    "        use_gcp_secret('user-gcp-sa'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_func = caption_pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_filename = pipeline_func.__name__ + '.pipeline.zip'\n",
+    "compiler.Compiler().compile(pipeline_func, pipeline_filename)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "client = kfp.Client()\n",
+    "experiment = client.create_experiment(EXPERIMENT_NAME)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h4> Run pipeline </h4>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Test run to make sure all parts of the pipeline are working properly\n",
+    "arguments = {\n",
+    "    'dataset_path': GCS_DATASET_PATH, \n",
+    "    'num_examples': 100, # Small test to make sure pipeline functions properly\n",
+    "    'training_batch_size': 16, # has to be smaller since only training on 80 examples \n",
+    "}\n",
+    "run_name = pipeline_func.__name__ + ' run'\n",
+    "run_result = client.run_pipeline(experiment.id, run_name, pipeline_filename,\n",
+    "                                params=arguments)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Model checkpoints are saved at training_output_dir, which is `[GCS_DATASET_PATH]/train/checkpoints/` by default."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h3> Steps taken to convert the original TF 2.0 notebook </h3>\n",
```

Comment:
suggestion: move these steps to the README.md since users will be interested to see these steps when they first check out the sample. When users actually read through/run the notebook, this paragraph will be less visible. WDYT?

Reply:
I agree, README seems like a better place for it.

Reply:
Great. Thanks